backend/hitas/views/building.py

```diff
@@ -17,6 +19,14 @@ class BuildingHitasAddressSerializer(serializers.Serializer):
 class BuildingSerializer(HitasModelSerializer):
     address = BuildingHitasAddressSerializer(source="*")
     building_identifier = ValueOrNullField(required=False, allow_null=True)
+    apartment_count = serializers.SerializerMethodField()
+
+    def get_apartment_count(self, instance: Building) -> int:
+        apartment_count: Optional[int] = getattr(instance, "apartment_count", None)
+        if apartment_count is not None:
+            return apartment_count
+
+        return instance.apartments.count()
```

Comment:
Wish we could just always annotate the `apartment_count` to the Building queryset in the REST API, but i don't think there is a simple enough way :/